libnd4j/tests_cpu/libnd4j_tests/CMakeLists.txt

```diff
@@ -19,30 +19,30 @@ if(result)
 endif()
 
 # OPTIONAL MKL-DNN
```

Comment:
Should updated this comments and filenames here too?

Reply:
Updated here as wlel as tests.